pkg/datastore/mock_unmarshaller.go

```diff
@@ -1,12 +1,11 @@
 // Code generated by MockGen. DO NOT EDIT.
```

Comment:
Please add a task to generate the appregistry mock files when the unit tests are run like you did for OperatorSource. We can then remove them from the repo.

Reply:
Done.